### ISSUE TITLE
Web - Step 5 - Incorrect Page Title

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#82]
The page TITLE should be "Step 5: Monitor Compliance - Comply Chain - United States Department of Labor" instead of only "Comply Chain". 

You should tell assistive technology users like screen reader users and search engines what the page is followed the application and then organization (DOL) and you could even add in ILAB spelled out before the DOL part.

The page TITLE format applies to all Comply Chain pages, so please fix accordingly.

Step 5 - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/5

**Comment by Abychalil** [comment]
title added as requested. Fixed

**Comment by jfc3-dol** [comment]
@Abychalil,

Thanks for correcting the screen TITLE. Will look at it when the next version/build is sent, and I  have time to test.

**Comment by jfc3-dol** [comment]
The page TITLE is now "Step 5: Monitor Compliance - Comply Chain - United States Department of Labor" which doesn't tell the user which Step they are on, but it's much better than only being "Comply Chain".

So I'm going to close the issue.
